assets/src/edit-story/components/form/color/test/getPreviewStyle.js

```diff
@@ -14,14 +14,29 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import {
+  createSolid,
+  generatePatternStyles as generatePatternStylesMock,
+} from '@web-stories-wp/patterns';
 /**
  * Internal dependencies
  */
-import generatePatternStylesMock from '../../../../utils/generatePatternStyles';
-import createSolid from '../../../../utils/createSolid';
 import getPreviewStyle from '../getPreviewStyle';
 
-jest.mock('../../../../utils/generatePatternStyles', () => jest.fn());
+jest.mock('@web-stories-wp/patterns', () => {
+  return {
+    generatePatternStyles: jest.fn(),
+    createSolid: (r, g, b, a = 1) => {
```

Comment:
We could just use `...jest.requireactual(...)` here (and elsewhere) as well so we don't have to mock `createSolid`